src/util/KeyboardNavigation.ts

```diff
@@ -0,0 +1,93 @@
+import { type ArrowKeyCode, KeyCode } from './KeyCode';
+import { arrayMinByKey, getActiveElement, isHTMLElement } from './CommonUtils';
+import { Rectangle } from './GeometryUtils';
+
+export function getFocusedChild(children: HTMLElement[]): HTMLElement | null {
```

Comment:
We only ever use the *first* element of this array. Perhaps we can have a `defaultFocusChild: HTMLElement | undefined` instead?

Also, should we have a way to pick a *different* default focus child that is *not* the first child? Some kind of `autofocus` attribute that customers can set on a child element?